indico/modules/rb_new/client/js/modules/bookRoom/BookRoom.jsx

```diff
@@ -148,6 +174,18 @@ class BookRoom extends React.Component {
                     </div>
                     {this.renderViewSwitch()}
                 </div>
+                <SearchResultCount matching={results.length} total={totalResultCount}
+                                   isFetching={isSearching} />
```

Comment:
Style comment: you may want to put `total={...}` in a new line to be consistent and more merge-friendly.